Doc/c-api/slice.rst

```diff
@@ -53,6 +53,20 @@ Slice Objects
 
    Returns ``0`` on success and ``-1`` on error with exception set.
 
+   .. note::
+      This function considered not safe for resizable sequences.  Replace its invocation ::
```

Comment:
Is there a typo?  Maybe *is considered*?  Also there is a whitespace before `::`.  I do not know if it is intentional, considering you apply the  `::` without a whitespace in the sentence below